roles/openshift_logging/tasks/main.yaml

```diff
@@ -37,6 +37,55 @@
   when:
     - not openshift_logging_install_logging | default(false) | bool
 
+- name: Create templates subdirectory
+  file:
+    state: directory
+    path: "{{ tempdir }}/templates"
+    mode: 0755
+  changed_when: False
+
+# storageclasses are used by default but if static then disable
+# storageclasses with the storageClassName set to "" in pvc.j2
+- name: Creating storage template - static
+  template:
+    src: pvc.j2
+    dest: "{{ tempdir }}/templates/{{ pvc_prefix }}-pvc.yml"
+  vars:
+    obj_name: "{{ openshift_logging_pvc_name }}"
+    size: "{{ openshift_logging_pvc_size }}"
+    access_modes: "{{ openshift_logging_pvc_access_modes | list }}"
+    pv_selector: "{{ openshift_logging_pv_selector }}"
```

Comment:
are we not going to differentiate between the pvs created for ES and the ones created for fluentd?